keyboards/acheron/apollo/88htsc/info.json

```diff
@@ -8,123 +8,217 @@
         "pid": "0x8873",
```

Comment:
Please run `qmk format-json` on all of these.

Reply:
Done in 0415331